src/components/claim/ClaimRewardsVeDF.svelte

```diff
@@ -75,112 +76,123 @@
+                ? parseFloat(
+                    $userBalances[
+                      getAddressByChainIdKey(
+                        process.env.VE_SUPPORTED_CHAINID,
+                        "veOCEAN"
+                      )
+                    ]
+                  ).toFixed(3)
+                : 0
+            } veOCEAN`,
+          },
+        ]}
+        loading={claiming === "VE_REWARDS"}
+        onClick={onClaimVeRewards}
+        disabled={canClaimVE === false ||
+          claiming !== undefined ||
+          $veClaimables <= 0}
+      />
+      <ClaimItem
+        title="Active"
+        apy={`${
+          $APYs
+            ? $APYs?.active > 10000
+              ? "over 10000"
+              : `${$APYs?.active.toFixed(2)}`
             : 0
-        } veOCEAN`,
-      },
-    ]}
-    loading={claiming === "VE_REWARDS"}
-    onClick={onClaimVeRewards}
-    disabled={canClaimVE === false ||
-      claiming !== undefined ||
-      $veClaimables <= 0}
-  />
-  <ClaimItem
-    title="Active"
-    apy={`${
-      $APYs
-        ? $APYs?.active > 10000
-          ? "over 10000"
-          : `${$APYs?.active.toFixed(2)}`
-        : 0
-    }% Avg APY ${
-      $userAddress
-        ? `| ${
-            $APYs
-              ? $APYs?.activeUser > 10000
-                ? "over 10000"
-                : `${$APYs?.activeUser.toFixed(2)}`
-              : 0
-          }% Your APY`
-        : ""
-    }`}
-    apyTooltip={descriptions.default.tooltip_rewards_apy_active}
-    description="<p>Earn Active Rewards from Data Farming by <strong>allocating veOCEAN</strong> and <strong>curating quality data</strong>.</p><p>Get <strong>2X Rewards by publishing</strong> your own datasets and allocating to them.</p>"
-    amount={`${parseFloat($dfClaimables).toFixed(3)} OCEAN`}
-    rewardTooltip={descriptions.default.tooltip_active_rewards}
-    metrics={[{ name: "allocated", value: `${$totalUserAllocation}%` }]}
-    showRedirectLink={(!$oceanUnlockDate || $totalUserAllocation <= 0) &&
-      $dfClaimables <= 0}
-    redirectLink={{ text: "Set allocations", url: "datafarming" }}
-    distributedAmount={roundInfo?.active}
-    loading={claiming === "DF_REWARDS"}
-    onClick={onClaimDfRewards}
-    disabled={canClaimDF === false ||
-      claiming !== undefined ||
-      $dfClaimables <= 0}
-    disableRedirect={!$oceanUnlockDate}
-  />
+        }% Avg APY ${
+          $userAddress
+            ? `| ${
+                $APYs
+                  ? $APYs?.activeUser > 10000
+                    ? "over 10000"
+                    : `${$APYs?.activeUser.toFixed(2)}`
+                  : 0
+              }% Your APY`
+            : ""
+        }`}
+        apyTooltip={descriptions.default.tooltip_rewards_apy_active}
+        description="<p>Earn Active Rewards from Data Farming by <strong>allocating veOCEAN</strong> and <strong>curating quality data</strong>.</p><p>Get <strong>2X Rewards by publishing</strong> your own datasets and allocating to them.</p>"
```

Comment:
```suggestion
        description="<p>Earn active rewards from Data Farming by <strong>allocating veOCEAN</strong> and <strong>curating quality data</strong>.</p><p>Get <strong>2X rewards by publishing</strong> your own datasets and allocating to them.</p>"
```

Reply:
We have capitalized these as Nouns because this is what the reward programs are called. I'm going to resolve this feedback as complete.